protocol/query_and_mutate.go

```diff
@@ -47,13 +47,19 @@ func NewObTableQueryAndMutate() *ObTableQueryAndMutate {
 
 func NewObTableQueryAndMutateWithParams(tableOperationType ObTableOperationType, rowKey []*table.Column, columns []*table.Column) (*ObTableQueryAndMutate, error) {
 	obTableBatchOperation := NewObTableBatchOperation()
-	obTableOperation, err := NewObTableOperationWithParams(tableOperationType, nil, columns)
-	if err != nil {
-		return nil, errors.WithMessage(err, "create table operation")
+	if tableOperationType == ObTableOperationInsert {
+		obTableOperation, err := NewObTableOperationWithParams(tableOperationType, rowKey, columns)
+		if err != nil {
+			return nil, errors.WithMessage(err, "create table operation")
+		}
+		obTableBatchOperation.SetObTableOperations([]*ObTableOperation{obTableOperation})
```

Comment:
obTableBatchOperation.SetObTableOperations([]*ObTableOperation{obTableOperation}) to be common line

Reply:
do not need to change